src/server/middleware/middleware.go

```diff
@@ -29,3 +29,49 @@ func WithMiddlewares(handler http.Handler, middlewares ...Middleware) http.Handl
 	}
 	return handler
 }
+
+// Skipper defines a function to skip middleware.
+// Returning true skips processing the middleware.
+type Skipper func(*http.Request) bool
+
+// DefaultSkipper returns false which processes the middleware.
+func DefaultSkipper(*http.Request) bool {
```

Comment:
If skipper is optional, we don't need a skipper always returns false.  
We just do not set the skipper